cli.js

```diff
@@ -38,7 +45,7 @@ Promise
 			});
 		}
 
-		return ui(cli.flags);
+		return ui(cli.flags, pkg);
```

Comment:
I think I'd rather have it added to the `options` object `Object.assign(cli.flags, {pkg})`.

But wait before doing that untill @sindresorhus replies. Most of the time, he tends to have a strong opinion on parameters :).